docs/_design/customize-hook.md

```diff
@@ -0,0 +1,67 @@
+### Controller changes
```

Comment:
For some reason this page doesn't show up in the [website preview](https://deploy-preview-142--metacontroller.netlify.com/design/). Maybe we just need to add the front matter at the top of the file:

```
---
title: Customize Hook
---
```

Reply:
Fixed :)